code/modules/mob/mob.dm

```diff
@@ -31,8 +31,19 @@
 	else
 		living_mob_list += src
 	prepare_huds()
+	initialize_emotes()
 	..()
 
+/mob/proc/initialize_emotes()
```

Comment:
Oh jesus why
You reference all emotes on all mobs for no reason
You don't even change them from being global singletons they're just references

Reply:
This was intentional, since each mob having a copy of each emote would've been a waste of memory. So they just have references instead